packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -31,6 +31,7 @@ export const DataTableComponent = <T extends RecordWithId>({
   noDataMessage,
   pagination,
   onChangePage,
+  onChangeSortBy,
```

Comment:
Passed the alternative `onChangeSortBy` down through these props, as passing it through "columns" had problems with stale closure (was capturing the state of searchParams when first instantiated and never updating)